### ISSUE TITLE
Exporting logs as ZIP results in a 0-byte file

### ISSUE THREAD

**Issue by Skaynod** [F0x1d/LogFox#148]
When exporting logs as a ZIP file, the resulting file has a size of 0 bytes. I am unable to open the file, as it appears to be empty or corrupted.

> **Steps I Took to Encounter the Issue:**

1. I selected the logs I wanted to export.

2. I chose the ZIP export option.

3. I started the export and downloaded the generated ZIP file.

4. I noticed that the downloaded ZIP file was 0 bytes and couldn’t be opened.

**Expected Behavior:**
The ZIP file should contain the exported logs and be accessible without issues.

**Current Behavior:**
The generated ZIP file is 0 bytes in size and cannot be opened.

> **Additional Information:**

Operating System: Android 10

Logfox Version: v2.0.3 d0ac7d6

Device Model: Xiaomi Redmi Note 9 Pro with MIUI 12.0.3

Root Status: Device is rooted with Magisk 28

> **Troubleshooting Attempts:**

- Restarted the app and tried exporting again

- Cleared the app cache and forced stop the app

- Uninstalled and reinstalled different versions of the app

- Restarted the phone and retried the export

- Attempted export with different log sets

![MiXplorer](https://github.com/user-attachments/assets/bdaa0311-7622-4b92-b91e-804122e3ce3d)

**Comment by F0x1d** [comment]
Do you mean these steps?

1. Select logs in LogFox tab
2. Click "To recording"
3. Export created recording as ZIP

**Comment by drogga** [comment]
Can it be the fact that is uses SAF, instead of storage R/W Permission(s) - IDK, but on systems that the "Files" (DocumentsUI) app is missing or disabled/frozen (the later I did on purpose to reproduce and should be addressed as an exception) I get no file at all, LogFox crashes instead, it's easily replaceable, so I didn't grab logs by for example using another app that does also capture crashes.

**Comment by F0x1d** [comment]
But the problem is "0 bytes files" and not the crash. But maybe it's also something with ROM and system Files app

**Comment by drogga** [comment]
> But maybe it's also something with ROM and system Files app

Exactly, IDT my comment is that off-topic, it's kind of on point, but I can delete it if you want...
___
Here's an off-topic thing, which I certainly have to delete:
The Build Actions Workflow https://github.com/F0x1d/LogFox/actions/runs/11671049563 throws 2 warnings:
`The following actions use a deprecated Node.js version and will be forced to run on node20: actions/checkout@v3.`
`The 'set-output' command is deprecated and will be disabled soon. Please upgrade to using Environment Files.`
The artifacts/files are only 4 days old and already expired, you might want to adjust that .yml for that one, the others seem fine ;)

**Comment by shuvashish76** [comment]
I've same issue with v2.0.6. Export or Zip both creates 0-bytes file if you use Files (`com.android.documentsui`), where as no files created if you share to 3rd party FM such as [Material Files](https://f-droid.org/en/packages/me.zhanghai.android.files/).

For now I'm using _Share_ feature of LogFox which does the work for me.

**Comment by ItzFlitz** [comment]
Same here using Export or Zip options it always leads to 0 bytes saved file.
